src/libraries/System.Text.Json/tests/Common/PropertyVisibilityTests.cs

```diff
@@ -1322,10 +1322,6 @@ public async Task JsonIgnoreAttribute()
         }
 
         [Fact]
-#if BUILDING_SOURCE_GENERATOR_TESTS
-        // Needs support for more collections.
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/53393")]
-#endif
```

Comment:
Were all of these previously-skipped tests in a passing state before this PR, or because of the fix in this PR? Either way, reenabling them now in the backport is goodness.

Reply:
They were all passing with the exception of one test that was testing the behaviour in question. I figured I should just turn them all on opportunistically.